src/client/dfuse/SConscript

```diff
@@ -62,13 +63,29 @@ def build_client_libs_shared(env, prereqs):
     il_obj = []
     for src in IOIL_SRC:
         il_obj += ilenv.SharedObject(os.path.join('il', src), SHOBJPREFIX='s_')
+
+    pil4dfsenv = env.Clone()
+    pil4dfsenv.AppendUnique(CFLAGS='-Wno-error=nonnull-compare')
+    if '-fvisibility=hidden' in pil4dfsenv['CFLAGS']:
+        pil4dfsenv['CFLAGS'].remove('-fvisibility=hidden')
+    pil4dfsenv.AppendUnique(LIBPATH=[Dir('../dfs')])
+    pil4dfsenv.AppendUnique(CPPPATH=[Dir('../dfs').srcnode()])
```

Comment:
given that much of this is the same as ioil, should we create a clone for common stuff so we don't have to repeat?

Reply:
I would prefer the current form to minimize the overlap with libioil since libioil could be not supported in future after libpil4dfs.so is mature enough IMO.